magento/magento2
# [MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Check a product to be added to the Order 
2. Check if "Select" checkbox is checked in the Products Grid
3. Click the "Add selected product(s) to order" button


### Resolved issues:
1. [x] resolves magento/magento2#31033: [MFTF] Refactoring of AdminCreateOrderAddProductCheckboxTest
